src/test_merge.py

```diff
@@ -0,0 +1,41 @@
+"""Test merge sort."""
+
+
+import pytest
+from random import randint
+
+
+to_sort = [
+    ([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]),
+    ([10, 9, 8, 7, 6, 5, 4, 3, 2, 1], [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]),
+    ([10, 9, 8, 17, 6, 300, 4, 3, 2, 11], [2, 3, 4, 6, 8, 9, 10, 11, 17, 300])
```

Comment:
You need more test cases that demonstrate the limits of merge sort. What if you've got duplicate values? What if you have pre-sorted regions? What if I through 100 sets of randomized numbers, do they all come out sorted?